JitsiParticipant.js

```diff
@@ -148,6 +148,16 @@ export default class JitsiParticipant {
         return this._id;
     }
 
+    /**
+     * Returns the XMPP identity. This is defined by your application in the
+     * JWT `context` claims section.
+     *
+     * @returns {object|undefined} - XMPP user identity.
+     */
+    getIdentity() {
+        return this._identity;
+    }
```

Comment:
This method name is very similar to `getId()`. Other ideas:

- `getAuthenticationContext()`
- `getXmppIdentity`
- `getRoomIdentity`
- `getJwtContext`

but those feel weirdly too specific. So I kept it simple.